test/__init__.py

```diff
@@ -751,6 +764,8 @@ def sanitize_cmd(cmd):
     cp.pop('$db', None)
     cp.pop('$readPreference', None)
     cp.pop('lsid', None)
+    # Versioned api parameters
+    cp.pop('apiVersion', None)
```

Comment:
If I recall, lot's of tests failed because of this. What do you think of:
```suggestion
    if MONGODB_API_VERSION:
        cp.pop('apiVersion', None)
```